pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -8416,7 +8416,7 @@ self: {
        libraryHaskellDepends = [ base bytestring unix ];
        librarySystemDepends = [ fuse ];
        preConfigure = ''
-         sed -i -e "s@  Extra-Lib-Dirs:         /usr/local/lib@  Extra-Lib-Dirs:         ${fuse}/lib@" HFuse.cabal
+         sed -i -e "s@  Extra-Lib-Dirs:         /usr/local/lib@  Extra-Lib-Dirs:         ${lib.getLib fuse}/lib@" HFuse.cabal
```

Comment:
This is a generated file and the change will be rolled back. The change needs to be made here: https://github.com/NixOS/cabal2nix/blob/master/cabal2nix/src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs#L243

Reply:
Filed https://github.com/NixOS/cabal2nix/pull/591

Reply:
As soon as aa0d300cfdf196cb19d86a4aa7a33dc2434ff7ab propagates to `staging` via `master`, this shouldn't be any issue anymore.